{"name": "Если авторизоваться под несуществующим пользователем, запрос возвращает ошибку", "status": "passed", "steps": [{"name": "Вход курьера в систему", "status": "passed", "parameters": [{"name": "credentials", "value": "{'login': 'Petrov890890', 'password': 'dfuebw7685'}"}], "start": 1705238702151, "stop": 1705238702527}], "start": 1705238702151, "stop": 1705238702527, "uuid": "f102ecf8-1697-4c83-8244-e7d02177e7a2", "historyId": "f963daf59a435eb7e4f89d6f73e370d2", "testCaseId": "f963daf59a435eb7e4f89d6f73e370d2", "fullName": "tests.test_courier_login.TestCourierLogin#test_authorization_with_non_existent_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "LAPTOP-1KK7H688"}, {"name": "thread", "value": "10292-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}